spark-terasort/terasort.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: "sparkoperator.k8s.io/v1beta2"
+kind: SparkApplication
+metadata:
+  name: terasort
+spec:
+  type: Scala
+  mode: cluster
+  image: mesosphere/spark:spark-3.0.0-hadoop-2.9-k8s-rc3
+  imagePullPolicy: IfNotPresent
+  mainClass: sorting.SortingApp
+  mainApplicationFile: "https://kudo-spark.s3-us-west-2.amazonaws.com/spark-scala-tests-3.0.0-20200819.jar"
+  arguments:
+    - <input bucket, ex s3a://terasort-data>>
+    - <output bucket, ex s3a://terasort-scratch/spark-terasort-sorted>
```

Comment:
Is there a better way to expose some of these parameters ( s3a.access.key/s3a.secret.key )?